src/main/java/org/folio/edge/rtac/RtacHandler.java

```diff
@@ -39,76 +41,207 @@
   private static final String PARAM_INSTANCE_ID = "instanceId";
   private static final String PARAM_INSTANCE_IDS = "instanceIds";
 
+  private static final String SUPPORTED_MIME_TYPE_PATTERN = "(text|\\*)\\s*/\\s*(xml|\\*)|(application|\\*)\\s*/\\s*(xml|json|\\*)";
```

Comment:
Is it possible explicitly list all the supported types?

Reply:
Currently by "all supported types" mentioned these types:
application/xml, text/xml, application/json  and */*. All these covered by this regexp. Do you mean that it's need to notice in description of this regular expression to mention these types?

Reply:
I meant to have a list with the MIME types instead of using a regexp.

Reply:
I agree with this feedback, I think it would be easier to understand if there was a set of specific mime types that were supported

Reply:
I added meaningful supported mime types definition for now.